README.md

```diff
@@ -1100,6 +1101,24 @@ If you're already using Python 3.7, switch the `language_version` accordingly. F
 `stable` is a tag that is pinned to the latest release on PyPI. If you'd rather run on
 master, this is also an option.
 
+## GitHub Actions
+
+Create a file named `.github/workflows/black.yml` inside your repository with:
+
+```yaml
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-python@v1
+      - uses: psf/black@stable
```

Comment:
Is this a tag we (the maintainers) have to manually do and keep up to date?

Reply:
Yes. Although, it is a branch*, one that will updated on every release anyway. 

*According to this issue, the stable tag is a now a branch: https://github.com/psf/black/issues/760

Reply:
It is, but it's something that's already being kept up to date for other reasons anyway. I believe `pre-commit` relies on it too.